deploy/README.md

```diff
@@ -1,5 +1,29 @@
 # Installation
 
+## Use kne_cli deploy with configuration yaml
+
+```bash
+kne_cli deploy deploy/kne/kind.yaml
+```
+
+The new deployment command will perform all the below operations in a single call based on a yaml file you can customize.
+
+* Deploy kind cluster
+  * name will be the name of the kind cluster
+  * recycle will allow an existing cluster to be deployed on
+  * version is the kind version
+  * image is the node image to pull
+* Install Metallb
+  * Based on manifests/metallb checked in configurations
+  * Read docker configuration and generate proper configuration file
+  * Wait for deployment to complete
+* Install Meshnet
+  * Based on manifests/meshnet check in configurations
+  * Wait for pods to be healthy
+
+From start to finish this will take ~1 min to bring up the cluster ready for you to deploy a topology to.
+This will stil need images for the devices you wish to use in the topology loaded into the cluster.
```

Comment:
Maybe add more details like:
---

After the deployment is finished additional steps might be required:

* import container images to the cluster image store (unless an image is available for pulling)
* install controllers for nodes which are managed externally

Reply:
I like to be fairly incremental on the PR's
This is already WAY overly massive of a PR
especially as I add unittests and refactor for that

loading images is also fairly expensive in time so would ideally only load the images needed for a topology so is likely a follow up PR to add that

Reply:
I meant only to provide the above mentioned section. Not the implementation or anything like that

Reply:
Meaning that importing images is not needed if the image is available in a repo that is accessible from the cluster (as it will be pulled by the controller)

And that installation of additional controllers might be required after the cluster is installed, given that controllers based operations is envisioned to be the preferred way of launching networking pods

Reply:
updated